pset-iterators.js

```diff
@@ -161,6 +187,10 @@ console.log('Problem 6:')
 
 // Add your code below this line
 
+const firstRepublican = presidents.find(president => (president.party === "Republican"))
+
+console.log(firstRepublican);
+
```

Comment:
@rlou88 Great job using the `find` method to return the first president in the `republican` party.

Take a look at this solution [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L175)